### ISSUE TITLE
client does not read proxy system properties

### ISSUE THREAD

**Issue by akshatprakash** [sypht-team/sypht-java-client#3]
the client should read http.proxyHost and http.proxyPort if available
more details: https://docs.oracle.com/javase/8/docs/technotes/guides/net/proxies.html

**Comment by simonmittag** [comment]
agreed, need to find a way to support SOCKS/TLS but not HTTP in the official client. I'll leave this open

**Comment by simonmittag** [comment]
PS: your contributor invite is still open. If you accept it we can work on a branch here, not a fork
